cmd/influx/main.go

```diff
@@ -108,6 +109,13 @@ Examples:
 	}
 	fs.Parse(os.Args[1:])
 
+	argsNotParsed := os.Args[len(os.Args)-fs.NArg():]
+	if len(argsNotParsed) > 0 {
```

Comment:
This block should use `flag.Args()` instead.

https://golang.org/pkg/flag/#Arg
> Args returns the non-flag command-line arguments.